pkgs/tools/misc/via/default.nix

```diff
@@ -28,7 +28,7 @@ appimageTools.wrapType2 {
     cp -r ${appimageContents}/usr/share/icons $out/share
 
     mkdir -p $out/etc/udev/rules.d
-    echo 'KERNEL=="hidraw*", SUBSYSTEM=="hidraw", MODE="0666", TAG+="uaccess", TAG+="udev-acl"' > $out/etc/udev/rules.d/92-viia.rules
+    echo 'KERNEL=="hidraw*", SUBSYSTEM=="hidraw", TAG+="uaccess", TAG+="udev-acl"' > $out/etc/udev/rules.d/92-viia.rules
```

Comment:
I read somewhere that TAG+="uaccess" is processed in some 70-....rules file, which means adding more uaccess rules in 92-viaa.rules won't work. (Why is the udev rule named "viaa" and not "via"?)